moduleroot/.travis.yml.erb

```diff
@@ -181,7 +182,7 @@ before_deploy:
 <% end -%>
 deploy:
   provider: puppetforge
-  user: <%= @configs['user'] %>
```

Comment:
`user` is preferred for V1 of the Deploy API
https://docs.travis-ci.com/user/deployment/puppetforge/

`username` is preferred for V2 of the Deploy API
https://docs.travis-ci.com/user/deployment-v2/providers/puppetforge/

Reply:
To my understanding, they have deprecated `user` in favour of `username`, but kept `user` as alias for backward compatibility to v1 deployer. The generated `.travis.yml` will currently use the v1 deployer, but enabled to switch to v2.